app/views/layouts/_nav.html.erb

```diff
@@ -29,4 +29,11 @@
       <%=t "Sign up" %>
     <% end %>
   <% end %>
+    <span class="d-sm-inline-block ml-4 border-right dropdown"><%=t "nav.language.title" %>
```

Comment:
I'd suggest using `http://primercss.io/archive/forms/#selects` for this

Reply:
Pls you do.